src/clj/rems/locales.clj

```diff
@@ -2,27 +2,37 @@
   {:ns-tracker/resource-deps ["translations/en.edn" "translations/fi.edn"]}
   (:require [clojure.java.io :as io]
             [mount.core :refer [defstate]]
-            [rems.config :refer [env]])
+            [rems.config :refer [env]]
+            [rems.util :refer [deep-merge]])
   (:import (java.io FileNotFoundException)))
 
-(defn- load-translation [language translations-directory]
-  (let [file (when translations-directory
-               (io/file translations-directory (str (name language) ".edn")))
-        resource-path (str "translations/" (name language) ".edn")
+(defn- translations-from-file [filename dir]
+  (let [file (when dir
+               (io/file dir filename))
+        resource-path (str dir filename)
```

Comment:
This is fragile. It will break if `dir` does not end with `/`. Also it's questionable that why the resource path should be parameterizable, because it is known at build time.

Reply:
Ok, changing it so that it is always taken from under theme.